client/index.html

```diff
@@ -9,7 +9,6 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="black">
     <meta name="description" content="NG6-Starter by @AngularClass">
     <link rel="icon" href="data:;base64,iVBORw0KGgo=">
-    <base href="/">
```

Comment:
In case of changing base URL it would be better to also change settings of browser-sync to make it behaves as close to production as possible. So I think it would be better not to use scripts for generating this.